app/src/main/java/org/wikipedia/editactionfeed/AddTitleDescriptionsActivity.kt

```diff
@@ -95,7 +84,21 @@ class AddTitleDescriptionsActivity : SingleFragmentActivity<AddTitleDescriptions
             AlertDialog.Builder(context)
                     .setCustomTitle(DialogTitleWithImage(context, R.string.description_edit_task_unlock_title, R.drawable.ic_illustration_description_edit_trophy, true))
                     .setMessage(R.string.description_edit_task_unlock_body)
-                    .setPositiveButton(R.string.title_description_get_started) { _, _ -> context.startActivity(AddTitleDescriptionsActivity.newIntent(context, EDIT_TASKS_TITLE_DESC_SOURCE)) }
+                    .setPositiveButton(R.string.onboarding_get_started) { _, _ -> context.startActivity(AddTitleDescriptionsActivity.newIntent(context, EDIT_FEED_TITLE_DESC)) }
+                    .setNegativeButton(R.string.onboarding_maybe_later, null)
+                    .show()
+        }
+
+        fun maybeShowTranslationEdit(context: Context) {
+            if (WikipediaApp.getInstance().language().appLanguageCodes.size < 2 || Prefs.getTotalUserDescriptionsEdited() < 2 || !Prefs.showEditActionTranslateDescriptionsUnlockedDialog()) {
```

Comment:
Keep thinking that the value of unlocking/showing the dialog should be set as a constant and put them into the Constants.java.

Reply:
Will do... but i guess 1 and 2 are allowed while linting..

Reply:
Done